config/src/main/resources/application.base.conf

```diff
@@ -31,6 +31,15 @@ akka {
       connecting-timeout = 2s
     }
   }
+  kafka {
+    producer {
+      kafka-clients {
+        acks = 1
+        delivery.timeout.ms = 60000 // 60 seconds
```

Comment:
actor timeout should be set to the same or longer value as this value in the prod configuration.